machine/quanta/quanta_bwde/kernel/config

```diff
@@ -0,0 +1,18 @@
+CONFIG_MODULES=y
+CONFIG_DEVTMPFS=y
+CONFIG_DEVTMPFS_MOUNT=y
```

Comment:
according to the kernel docs, with DEVTMPFS_MOUNT=y, the devtmpfs filesystem is automounted by the kernel on /dev right after the initramfs rootfs is mounted.  there should be no need to mount it manually, like what you have below.